ext/curl/multi.c

```diff
@@ -564,8 +564,17 @@ static int _php_curl_multi_setopt(php_curlm *mh, zend_long option, zval *zvalue,
 		case CURLMOPT_MAX_PIPELINE_LENGTH:
 		case CURLMOPT_MAX_TOTAL_CONNECTIONS:
 #endif
-			error = curl_multi_setopt(mh->multi, option, zval_get_long(zvalue));
+		{
+			zend_long lval = zval_get_long(zvalue);
+
+#if LIBCURL_VERSION_NUM >= 0x073e00 /* 7.62.0 */
```

Comment:
I think we should throw this deprecation warning independent of the libcurl version. Or possibly throw a deprecation on old versions and an actual warning if it's no longer supported.

Reply:
Good idea. Thanks! I went with the second option.